support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/__tests__/paymentSelectionTests.js

```diff
@@ -0,0 +1,148 @@
+// flow
+
+import {
+  getProductOptions,
+  getCurrencySymbol,
+  getDisplayPrice,
+  getProductPrice,
+  getSavingPercentage,
+  mapStateToProps,
+} from '../helpers/paymentSelection';
+
+jest.mock('ophan', () => {}); // required to get the test to run! why?
```

Comment:
we had trouble with ophan firing when it was imported, I'm sure a fix went in to prevent that, if it was @rupertbates who made the change, but I'm not sure if that was pulled down.
If it's still breaking tests that sounds like there might still be a side effect somewhere we don't expect.